serversideapi/src/main/java/com/example/serversideapi/ServersideapiApplication.java

```diff
@@ -3,6 +3,7 @@
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 
+// Main class of api for execution
```

Comment:
Really? Who doesn't know that `@SpringBootApplication` is "Main class of api for execution"

Reply:
Thanks for the feedback.
I'll change my comments structure